docs/rewards/reward-boosts.mdx

```diff
@@ -0,0 +1,67 @@
+---
+id: reward-boosts
+title: Rewards Boosts
+sidebar_label: Rewards Boosts
+slug: /project/rewards-boosts
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+## Overview
+In addition to the **Base Rewards**, a device already gets for contributing to weather data during the regular daily distribution. Learn more about the daily rewards [Reward Mechanism (RM) ](/reward-mechanism).
+
+**Reward Boosts** rewards devices with extra $WXM tokens for meeting specific criteria. 
+
+So far the following **Reward Boosts** are implemented:
+
+### Beta Rewards
+
+According to the [Token Allocation table](https://weatherxm.network/docs/wxm-token.html#token-allocation) 3M $WXM are reserved to reward stations that participated to the network before the Token Launch Day ("Beta period").
+
+Beta rewards distribution will last 425 days, with Day 1 being the Token Launch day.
+
+Calculation:
+
+The following calculations will take place for each station, the day before the Token Launch:
+
+`RewardableBetaHours[s]`.
+This is the total number of hours the station provided valid data during the Beta period.
+A rewardable hour is an hour during which the station sent at least one valid weather data point.
+
+`StationBetaRewards[s] = 3000000 * RewardableBetaHours[s]/Sum(RewardableBetaHours of all stations)`
+This is the percentage of the 3m tokens that correspond to the rewardable hours the station provided.
+`DailyBetaRewards[s] = StationBetaRewards[s]/365`
+
+This is the beta rewards allocated daily to this station.
```

Comment:
Please move those in a new line and do the following:

`DailyBetaRewards[s] = StationBetaRewards[s]/365`: This is the beta rewards allocated daily to this station.